backend/models.py

```diff
@@ -0,0 +1,55 @@
+
+# Create your models here.
+from rest_framework.authtoken.models import Token
+from django.contrib.auth.models import AbstractUser
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+import uuid
+
+
+class User(AbstractUser):
+    uuid = models.UUIDField(
+        primary_key=True, uuid=uuid.uuid4, editable=False, unique=True)
+    # Using: username, password, first_name, last_name, email inherited from Abstractuser
```

Comment:
What do we need uuid here for? Why can't we just use the django model auto generated id to represent each user? It's more expensive to include the uuid when we do query?

Reply:
Also I think you can't pass `uuid` as  an argument to `models.UUIDField`, it should be `default` instead

Reply:
You are correct. We don't need the UUID field. I removed it entirely.

Reply:
I am thinking we can also add a Github Url  field for each user